.freeCodeCamp/client/components/heading.tsx

```diff
@@ -46,6 +48,7 @@ export const Heading = ({
       {goToNextLesson && (
         <button
           className='next-lesson-btn'
+          disabled={!canGoForward}
```

Comment:
This wasn't working for me - it was making them always disabled - so I just went with the cursor styles. I'll give this a test and see if it's working.